src/server/metrics.rs

```diff
@@ -600,3 +608,16 @@ pub fn record_request_source_metrics(source: String, duration: Duration) {
         }
     });
 }
+
+impl From<u32> for ResourcePriority {
+    fn from(priority: u32) -> Self {
+        // the mapping definition of priority in TIDB repo,
+        // see: https://github.com/bufferflies/tidb/blob/8b151114546d6a02d8250787a2a3213620e30524/parser/parser.y#L1740-L1752
+        match priority {
```

Comment:
```suggestion
        // see: https://github.com/pingcap/tidb/blob/.../parser/parser.y#L1740-L1752
```
please change to pingcap

Reply:
fixed